src/Browscap/Generator/Helper/BuildHelper.php

```diff
@@ -160,7 +160,9 @@ public static function run(
 
                 foreach (array_keys($sectionsWithVersion) as $sectionName) {
                     if (array_key_exists($sectionName, $output)) {
-                        $logger->error('tried to add section "' . $sectionName . '" more than once -> skipped');
+                        $logger->error(
+                            'tried to add section "' . $sectionName . '" from "' . $division->getName() . '" more than once -> skipped'
```

Comment:
I added this to help find what pattern was causing this error, figured it wouldn't hurt to keep it.